### PR TITLE
Vectorize pdeeqs

### PR DESCRIPTION
**Pull request by xtalax**
A one line update enforcing that eqs is a vector

### CODE REVIEW

**Comment by xtalax** [comment]
Also updated docs to point to MethodOfLines

File: docs/src/index.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ system:
     - Multi-package interface for numerical solving `OptimizationSystem`
 - [NeuralPDE.jl](https://github.com/SciML/NeuralPDE.jl)
     - Physics-Informed Neural Network (PINN) training on `PDESystem`
-- [DiffEqOperators.jl](https://github.com/SciML/DiffEqOperators.jl)
+- [MethoodOfLines.jl](https://github.com/SciML/MethodOfLines.jl)
```
Comment (baggepinnen) [inline docs/src/index.md:139]:
```suggestion
- [MethodOfLines.jl](https://github.com/SciML/MethodOfLines.jl)
```

Comment (xtalax) [reply docs/src/index.md:139]: Thanks!

**Comment by ChrisRackauckas** [comment]
@KirillZubov I assume there's no NeuralPDE.jl performance hit to assuming this?

**Comment by KirillZubov** [comment]
it won't affect
but this is no longer needed in NeuralPDE
https://github.com/SciML/NeuralPDE.jl/blob/a5bfc91094861deaf35383273967c4cc24c618ce/src/pinns_pde_solve.jl#L1059-L1061

**Pull request merged**